config/owasp/suppressions.xml

```diff
@@ -21,6 +23,7 @@
       CVE-2017-10355 https://github.com/jeremylong/DependencyCheck/issues/4614 (xercesImpl - may be false positive)
       CVE-2020-5408
       CVE-2021-37533 https://nvd.nist.gov/vuln/detail/CVE-2021-37533 (apache commons)
+      CVE-2021-4277  https://nvd.nist.gov/vuln/detail/CVE-2021-4277
       CVE-2022-1471  https://github.com/google/security-research/security/advisories/GHSA-mjmj-j48q-9wg2 (SnakeYaml)
       CVE-2022-40152 https://github.com/advisories/GHSA-3f7h-mf4q-vrm4 (woodstox-core - may be false positive)
       CVE-2022-41915 https://github.com/netty/netty/security/advisories/GHSA-hh82-3pmq-7frp (netty-codec)
```

Comment:
The <notes> section doesn't contain lines for all of the CVEs that follow it.  Can you add the following to the <notes> section: CVE-2020-5412, CVE-2022-22976, CVE-2022-38752 and CVE-2022-41881.  Make sure the order matches that of the CVEs.

Reply:
Missing CVEs added to notes section.